### PR TITLE
Render board

### PR DESCRIPTION
**Pull request by GDemps**
I removed some of the reveal conditions due to the reveal only applying to the "S".   Render board method complete and passing all tests.

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -25,15 +25,15 @@ def fire_upon
   end
 
   def render(reveal=false)
-    if !empty? && fired_upon? && @ship.sunk? || (!empty? && fired_upon? && @ship.sunk?) && reveal == true
```
Comment (GDemps) [inline lib/cell.rb:28]: If ship is sunk both players will see the "X"s.

File: lib/cell.rb [hunk 2]
```diff
@@ -25,15 +25,15 @@ def fire_upon
...
+    if !empty? && fired_upon? && @ship.sunk?
       "X"
-    elsif !empty? && fired_upon? || (!empty? && fired_upon?) && reveal == true
+    elsif !empty? && fired_upon?
```
Comment (GDemps) [inline lib/cell.rb:30]: if ship is hit both players will see the "H"s.

File: lib/board.rb [hunk 3]
```diff
@@ -73,4 +73,16 @@ def is_overlap?(coord)
...
+    rendered_cell = ""
+    cells.each do |cell|
+       rendered_cell += cell[1].render(reveal) + " "
+    end
+    "  1 2 3 4 \n" +
+    "A #{rendered_cell[0..7]}\n" +
+    "B #{rendered_cell[8..15]}\n" +
+    "C #{rendered_cell[16..23]}\n" +
+    "D #{rendered_cell[24..31]}\n"
+  end
```
Comment (ajtran303) [inline lib/board.rb:86]: Nice, I ran the tests and it checks out.

**Review by ajtran303 — APPROVED** [review]
Nice job. I figured out how to `checkout` a remote branch and ran the tests. Approved.

**Pull request merged**